src/wp-includes/template-loader.php

```diff
@@ -27,58 +27,79 @@
 }
 
 // Process feeds and trackbacks even if not using themes.
-if ( is_robots() ) :
+if ( is_robots() ) {
 	/**
 	 * Fired when the template loader determines a robots.txt request.
 	 *
 	 * @since WP-2.1.0
 	 */
 	do_action( 'do_robots' );
 	return;
-elseif ( is_feed() ) :
+} elseif ( is_feed() ) {
 	do_feed();
 	return;
-elseif ( is_trackback() ) :
-	include ABSPATH . 'wp-trackback.php';
+} elseif ( is_trackback() ) {
+	include( ABSPATH . 'wp-trackback.php' );
 	return;
-endif;
+}
+
+if ( defined( 'WP_USE_THEMES' ) && WP_USE_THEMES ) {
+
+	$tag_templates = array(
+		'is_embed'             => 'get_embed_template',
+		'is_404'               => 'get_404_template',
+		'is_search'            => 'get_search_template',
+		'is_front_page'        => 'get_front_page_template',
+		'is_home'              => 'get_home_template',
+		'is_privacy_policy'    => 'get_privacy_policy_template',
+		'is_post_type_archive' => 'get_post_type_archive_template',
+		'is_tax'               => 'get_taxonomy_template',
+		'is_attachment'        => 'get_attachment_template',
+		'is_single'            => 'get_single_template',
+		'is_page'              => 'get_page_template',
+		'is_singular'          => 'get_singular_template',
+		'is_category'          => 'get_category_template',
+		'is_tag'               => 'get_tag_template',
+		'is_author'            => 'get_author_template',
+		'is_date'              => 'get_date_template',
+		'is_archive'           => 'get_archive_template',
+	);
+	$template      = false;
+
+	// Loop through each of the template conditionals, and find the appropriate template file.
+	foreach ( $tag_templates as $tag => $template_getter ) {
+		if ( call_user_func( $tag ) ) {
+			$template = call_user_func( $template_getter );
+		}
+
+		if ( $template ) {
+			if ( 'is_attachment' === $tag ) {
+				remove_filter( 'the_content', 'prepend_attachment' );
+			}
```

Comment:
`is_privacy_policy` and `get_privacy_policy_template` are since WP-5.2.
The others are OK.